build.gradle.kts

```diff
@@ -54,6 +54,7 @@ extra.apply {
     set("kafkaVersion", "2.7.0")
     set("gson", "2.13.1")
     set("jackson", "2.19.1")
+    set("dropwizardMetrics", "5.0.0")
```

Comment:
Let's validate that Confluent is not using this package. We might need to shade it